packages/lib/gpc/src/gpcTypes.ts

```diff
@@ -252,8 +289,12 @@ export type GPCProofInputs = {
    */
   owner?: GPCProofOwnerInputs;
 
-  // TODO(POD-P2): List membership configuration
-  // TODO(POD-P2): Tuple configuration
+  /*
+   * Named lists of valid values for each list membership check. These values
+   * may be primitive (i.e. of type PODValue) or tuples (represented as
+   * PODValueTuple = PODValue[]).  Each list must contain at least two elements.
```

Comment:
I took this out of the *MVP GPC Details* section of the spec. I'll drop this condition.

Reply:
Good callout.  I think my intent when I wrote that is that a circuit should support lists with size >= 2, though given padding a user can use it to hold only one value.
That's mostly a degenerate case regardless, since membership in a list of 1 value is equivalent to revealing the specific value.  But I think we should allow it just so that users don't have to worry about the edge case.